docs/features/configurationReload.md

```diff
@@ -29,3 +29,4 @@ reload-jcasc-configuration
   import io.jenkins.plugins.casc.ConfigurationAsCode;
   ConfigurationAsCode.get().configure()
   ```
+  Note that running the above code in a declarative pipeline (either directly or via shared libraries) will put this plugin in a bad state where the configuration cannot be reload at all until Jenkins is restarted. See #1227 for more info.
```

Comment:
is it better to just remove the above from the docs?

Reply:
Ultimately that is something for CloudBees/CJCasC maintainers to decide, but I don't believe so. So far we've been using this successfully as per the [workaround posted](https://github.com/jenkinsci/configuration-as-code-plugin/issues/1227#issuecomment-566470115) in #1227, and this works as long as a) the script approvals allow it, b) it's a freestyle job, and c) it is run using `systemGroovyCommand`. The issue of course is when this is run through pipelineJob. Mind you, even if the above three predicates aren't met and the code is run in a freestyle job, Jenkins is still in a good state and the code above can be run e.g. within the script console fine.